src/local_instance.py

```diff
@@ -2,24 +2,24 @@
 import subprocess, os, threading
 from time import sleep
 from logging import getLogger
+from .internal import data_dir, cache_dir
 
 
 logger = getLogger(__name__)
 
 instance = None
 port = 11435
-data_dir = os.getenv("XDG_DATA_HOME")
 overrides = {}
 
 def start():
-    if not os.path.isdir(os.path.join(os.getenv("XDG_CACHE_HOME"), 'tmp/ollama')):
-        os.mkdir(os.path.join(os.getenv("XDG_CACHE_HOME"), 'tmp/ollama'))
+    if not os.path.isdir(os.path.join(cache_dir, 'tmp/ollama')):
+        os.mkdir(os.path.join(cache_dir, 'tmp/ollama'))
     global instance, overrides
     params = overrides.copy()
     params["OLLAMA_HOST"] = f"127.0.0.1:{port}" # You can't change this directly sorry :3
     params["HOME"] = data_dir
-    params["TMPDIR"] = os.path.join(os.getenv("XDG_CACHE_HOME"), 'tmp/ollama')
-    instance = subprocess.Popen(["/app/bin/ollama", "serve"], env={**os.environ, **params}, stderr=subprocess.PIPE, text=True)
```

Comment:
Not very sure if `/app/bin` is in default `PATH` inside flatpak, but should be

Reply:
yeah this is $PATH /app/bin:/usr/bin